### PR TITLE
Bridge 1226 Profile 

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Includes a stubbed out profile view controller and code to handle withdrawing from a study.

### CODE REVIEW

File: BridgeAppSDK/Resources/Base.lproj/Withdraw.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "taskIdentifier"            : "withdraw",
+    "steps":
+    [
+     {
+     "identifier"       : "withdraw",
+     "title"            : "Withdrawal Survey",
+     "text"             : "Please help us make our study better by explaining the reasons for your withdrawl.",
```
Comment (Erin-Mounts) [inline BridgeAppSDK/Resources/Base.lproj/Withdraw.json]: withdraw**a**l

File: BridgeAppSDK/SBAAppDelegate.swift [hunk 2]
```diff
@@ -35,19 +35,41 @@ import UIKit
...
 @objc
-public protocol SBAAppInfoDelegate: class {
+public protocol SBAAppInfoDelegate: NSObjectProtocol {
     var currentUser: SBAUserWrapper { get }
     var bridgeInfo: SBABridgeInfo { get }
+}
+
+/**
+ The BridgeAppSDK delegate is used to define additional functionality on the app delegate.
+ It is used throughout this SDK with the assumtion that the UIAppDelegate will conform to
```
Comment (Erin-Mounts) [inline BridgeAppSDK/SBAAppDelegate.swift]: assum**p**tion

File: BridgeAppSDK/SBAUserWrapper+Bridge.swift [hunk 3]
```diff
@@ -303,6 +306,100 @@ public extension SBAUserWrapper {
...
+        }))
+        
+        appDelegate.presentViewController(controller, animated: true) { }
+    }
+    
+    fileprivate func promptForWithdraw() {
+        guard let json = SBAResourceFinder.shared.json(forResource: "Withdraw"),
+            let task = (json as NSDictionary).createORKTask()
+        else {
+            assertionFailure("Failed to create withdrawl survey")
```
Comment (Erin-Mounts) [inline BridgeAppSDK/SBAUserWrapper+Bridge.swift]: withdraw**a**l

**Review by Erin-Mounts — COMMENTED** [review]
just a couple of notes about typos.

File: BridgeAppSDK/Resources/Base.lproj/Withdraw.json (hunk 1 above)
Comment (syoung-smallwisdom) [reply BridgeAppSDK/Resources/Base.lproj/Withdraw.json]: That is what the Parkinson app says. I thought it was intentional. If you think about it "Withdraw Survey" doesn't sound right.

Comment (Erin-Mounts) [reply BridgeAppSDK/Resources/Base.lproj/Withdraw.json]: withdrawl --> withdraw**a**l is what I meant

File: BridgeAppSDK/Base.lproj/BridgeAppSDK.strings [hunk 4]
```diff
@@ -133,6 +133,7 @@
 "SBA_SURVEY_LOADING_TEXT" = "Please wait while the survey is downloaded."; // Message for a loading a survey.
 "SBA_COPYRIGHT" = "Copyright"; // Text on the copyright button.
 
-
-
+/* Withdrawl */
```
Comment (Erin-Mounts) [inline BridgeAppSDK/Base.lproj/BridgeAppSDK.strings]: sorry, missed this one the first time through

File: BridgeAppSDK.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -186,6 +185,7 @@
 		FF92A6791DBFF13700614D1F /* Instructions.html in Resources */ = {isa = PBXBuildFile; fileRef = FF92A67B1DBFF13700614D1F /* Instructions.html */; };
 		FF92A67F1DBFF14000614D1F /* Withdrawl.html in Resources */ = {isa = PBXBuildFile; fileRef = FF92A6811DBFF14000614D1F /* Withdrawl.html */; };
```
Comment (Erin-Mounts) [inline BridgeAppSDK.xcodeproj/project.pbxproj]: apparently this has been like this all along... not important to fix unless you feel like it

**Review by Erin-Mounts — APPROVED** [review]
found one more instance of the "withdrawl" spelling (in a comment in the .strings file) and it seems there's a Withdrawl.html file that's been spelled like that all along--anyway fix if you want, merge when ready

**Pull request merged**